manifests/init.pp

```diff
@@ -23,8 +26,9 @@
 #   to 'u_puppet_environment'.
 class servicenow_cmdb_integration (
   String $instance,
-  String $user,
-  String $password,
+  String $user              = 'default_user',
```

Comment:
Ah sorry, I think it's enough to make `user`, `password`, `oauth_token` optional. Then they'll automatically default to `undef` if not specified. That way, this doesn't have to check for hard-coded values and the script doesn't have to either. Script can just check `.nil?` like it used to b/c `undef` is interpreted as `nil` in Ruby.

Reply:
I think you already did that in an earlier iteration actually, so why revert it?

Reply:
Reverting back. Sorry I misunderstood some of your previous comments.